lib/build.api.js

```diff
@@ -70,7 +69,7 @@ class Build {
       let project = this.aquifer.project;
 
       // Set some reasonable defaults.
-      this.options.makeFile = this.getAbsolutePath(this.options.makeFile) || project.absolutePaths.makeFile;
+      this.options.makeFile = project.absolutePaths.make;
```

Comment:
@ccjjmartin we probably still want to accept a command option here. I think the problem may be that `this.getAbsolutePath()` is always returning a value even if `this.options.makeFile` is empty. If we replaced this with the following ternary would that also resolve the issue?

```
this.options.makeFile = this.options.makeFile ? this.getAbsolutePath(this.options.makeFile) : project.absolutePaths.makeFile;
```